### PR TITLE
Pin @radix-ui/react-id dependency to 0.1.1

### PR DESCRIPTION
**Pull request by hvergara**
*Issue #, if available:*
#967 
https://github.com/radix-ui/primitives/issues/1026

*Description of changes:*
One of our dependencies (`@radix-ui/react-id`) stopped exporting the `IdProvider` symbol in version `0.1.2` causing errors on customers trying to build their projects. This PR locks the dependency verison to `0.1.1` until we have a fix for this issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by alharris-at — APPROVED** [review]

File: packages/react/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9",
     "@radix-ui/react-accordion": "^0.1.1",
     "@radix-ui/react-dropdown-menu": "^0.1.1",
-    "@radix-ui/react-id": "^0.1.0",
+    "@radix-ui/react-id": "0.1.1",
     "@radix-ui/react-slider": "^0.1.1",
     "@radix-ui/react-tabs": "^0.1.1",
```
Comment (ericclemmons) [inline packages/react/package.json:57]: It may be best to pin _all_ `@radix-ui/*` dependencies given today's scenario.

Comment (hvergara) [reply packages/react/package.json:57]: good call, preparing another commit

Comment (hvergara) [reply packages/react/package.json]: done

**Review by ericclemmons — APPROVED** [review]

**Review by reesscot — APPROVED** [review]
LGTM

**Pull request merged**